ivy/functional/frontends/numpy/mathematical_functions/sums_products_differences.py

```diff
@@ -0,0 +1,25 @@
+# global
+import ivy
+
+# local
+from ivy.func_wrapper import from_zero_dim_arrays_to_float
+
+
+@from_zero_dim_arrays_to_float
+def prod(
```

Comment:
Here you are supposed to recreate the input arguments of numpy.prod, which is shown here: https://numpy.org/doc/stable/reference/generated/numpy.prod.html
So the argument name and initial value needs to be as similar as possible without relying on the content of numpy framework.

Reply:
In that case I understood task correctly, but I am still having troubles with this part, because I am not sure which initial values I need to use instead of `np._NoValue()`. My current option is 
```
keepdims=False,
initinal=None,
where=True
```
but it is not working.

Reply:
Well, this is a good question. My personal guess is that they all should be initially None. But I am asking other professional people in our team about this. Please be patient, thank you.

Reply:
I just got a confimation, you should use None in this case.